src/utils/web.ts

```diff
@@ -10,6 +15,36 @@ export const getField = (req: Request, fieldName: string): string => {
   return "";
 };
 
+/**
+ * Set the back link for a page. 
+ * Checks whether the user came from the check your answers page and if so,
+ *  sets the back link to the check your answers page instead.
+ */
+export const setBackLink = (req: Request, checkYourAnswersLink: string | undefined, backLink: string): string => {
+  if(checkYourAnswersLink){
+    return checkYourAnswersLink;
+  }
+  return backLink;
+};
+
+/**
+ * Set the redirect link for a page. 
+ * Checks whether the user came from the check your answers page and if so,
+ *  sets the redirect link to the check your answers page instead.
+ */
+export const setRedirectLink = async (req: Request, checkYourAnswersLink: string | undefined, backLink: string): Promise<string> => {
```

Comment:
should this variable name backLink be redirectLink?
just pointing it out as I had a hard time understanding it in controller classes.

Reply:
Yeah good point, must have copied the method and not updated.